week2/2-website/css/style.css

```diff
@@ -3,9 +3,224 @@
 
 body {
     font-family: 'Roboto', sans-serif;
+    font-weight: 120;
     -webkit-font-smoothing: antialiased;
+    /*border: 1px solid red;*/
+    
+    
 }
 
+.nav-header {
+    display: flex;
+    flex-direction: row;
+    background-color: white;
+    
+    padding: 0.5rem;
+    max-height: 4rem;
+    position: fixed;
+    width: 100%;
+    border-bottom: 1px solid rgb(218, 218, 218);
+    
+}
+
+.nav-links {
+    display: flex;
+    flex-direction: row;
+    /*border: 1px solid purple;*/
+    width: 60%;
+    color: rgb(148, 147, 147);
+    font-weight: 550;
+    font-size: 1rem;
+    
+}
+
+li:nth-child(1) {
+    color: rgb(78, 75, 75);
+}
+
+
+ul {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    /*border: 1px solid blue;*/  
+    width: 100%;
+}
+
+.logo {
+    /*border: 1px solid black;*/
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    flex-grow: 4;
+    width: 30%  
+
+}
+
+li {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding-left: 1.0rem;
+    padding-right: 1.0rem;
+    
+    /*border: 1px solid yellow;*/    
+}
+.image-container {
+    display: flex;
+    flex-direction: column;
+    
+    justify-content: center;
+    /*border: 1px solid red;*/
+}
+.image-section {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    background-image: url("/img/first-background.jpg ");
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: cover;
+    color: white;
+    height: 42rem;
+    
+}  
+.image-container p {
+    display: flex;
```

Comment:
None of these attributes seem to be doing anything. 
You generally wouldn't be giving `text` tags a display of flex.
I think you can remove this block altogether.
Also FYI - It's good practice to remove any commented code before submitting a PR